packages/manager/modules/overthebox/src/overthebox/autoconfigure/translations/Messages_fr_FR.json

```diff
@@ -30,5 +30,40 @@
   "overTheBox_autoconfigure_create_failed": "Une erreur s'est produite lors de la création : {{errorMessage}}.",
   "overTheBox_autoconfigure_remove_failed": "Une erreur s'est produite lors de la suppression : {{errorMessage}}.",
   "overTheBox_autoconfigure_mandatory": "Ce champ est obligatoire",
-  "overTheBox_autoconfigure_actions": "Actions"
+  "overTheBox_autoconfigure_actions": "Actions",
+  "overTheBox_autoconfigure_firewall_title": "Firewall",
+  "overTheBox_autoconfigure_firewall_select_value": "Sélectionnez une valeur...",
+  "overTheBox_autoconfigure_firewall_rule_create_succeed": "La création de la règle a été effectuée avec succès.",
+  "overTheBox_autoconfigure_firewall_rule_remove_succeed": "La suppression de la règle a été effectuée avec succès.",
+  "overTheBox_autoconfigure_firewall_redirect_create_succeed": "La création de la redirection a été effectuée avec succès.",
+  "overTheBox_autoconfigure_firewall_redirect_remove_succeed": "La suppression de la redirection a été effectuée avec succès.",
+  "overTheBox_autoconfigure_firewall_rules": "Règles du firewall",
+  "overTheBox_autoconfigure_firewall_source_zone": "Zone source",
+  "overTheBox_autoconfigure_firewall_target": "Cible",
+  "overTheBox_autoconfigure_firewall_target_optional": "Cible (optionel)",
```

Comment:
**issue(blocking):** There are some typo (`optionel` must be replaced by `optionnel`).